app/src/main/java/io/homeassistant/companion/android/webview/WebViewPresenterImpl.kt

```diff
@@ -25,8 +25,8 @@ class WebViewPresenterImpl @Inject constructor(
 
     override fun onViewReady() {
         mainScope.launch {
-            val url = integrationUseCase.getUiUrl(false)
-                ?: throw IllegalStateException("Unable to display the webview if we do not have url")
+            val ssid = view.getCurrentSsid()
+            val url = urlUseCase.getUrl("\"$ssid\"" == urlUseCase.getHomeWifiSsid())
```

Comment:
Why the extra quotes ?

Reply:
https://developer.android.com/reference/android/net/wifi/WifiInfo.html#getSSID()

Reply:
But I have my logic reversed...